app/models/user.rb

```diff
@@ -1,4 +1,8 @@
-class User < ActiveRecord::Base
+qclass User < ActiveRecord::Base
```

Comment:
whoops 

----- Original Message -----

From: "Levi Cole" <notifications@github.com> 
To: "TeriPastorino/KnoWater" <KnoWater@noreply.github.com> 
Cc: "TeriPastorino" <tpasto@comcast.net> 
Sent: Wednesday, May 20, 2015 8:40:25 AM 
Subject: Re: [KnoWater] Shower log (#8) 



In app/models/user.rb : 
> @@ -1,4 +1,8 @@
> -class User < ActiveRecord::Base
> +qclass User < ActiveRecord::Base 


typo here 

— 
Reply to this email directly or view it on GitHub .